layers/+distributions/spacemacs-bootstrap/packages.el

```diff
@@ -342,14 +342,40 @@
       (push (cons (cons nil (concat "\\`" (car nd) "\\'")) (cons nil (cdr nd)))
             which-key-replacement-alist)))
 
-  (push '(("\\(.*\\) 0" . "select-window-0") . ("\\1 0..9" . "window 0..9"))
+  ;; Group together sequence and identical key entries in the which-key popup.
+  ;; SPC- Spacemacs root
+  ;; rename "0 -> winum-select-window-0-or-10"
+  ;; to  "0..9 -> winum-select-window-0..9-or-10"
+  (push '(("\\(.*\\) 0" . "winum-select-window-0-or-10") .
```

Comment:
Would it be better to move the `.` from the end of the line to the beginning of the next line?
There are 3 other occurrences below.